lib/agent0/agent0/base/config/environment_config.py

```diff
@@ -26,7 +26,7 @@ class EnvironmentConfig(FrozenClass):
     # if halt_on_errors is True, halt_on_slippage controls if we halt when slippage happens
     halt_on_slippage: bool = False
     # optional output filename for logging
-    log_filename: str = "agent0-logs"
+    log_filename: str = ".logging/agent0_logs.log"
```

Comment:
im surprised you need this. I was under the impression that `.logging/` and `.log` were added automatically by the logging lib. Did you verify that this is putting the log into the right spot?

Reply:
Verified. Without the prefix, logging goes into the ether and is lost forever